lua/luasnip/util/extend_decorator.lua

```diff
@@ -8,8 +11,8 @@ local function default_extend(arg, extend)
 end
 
 ---Create a new decorated version of `fn`.
----@param fn The function to create a decorator for.
----@vararg The values to extend with. These should match the descriptions passed
+----@param fn The function to create a decorator for.
+----@vararg The values to extend with. These should match the descriptions passed
```

Comment:
Oh yeah, I don't know where comes from. Maybe because lua-ls was hinting an error? lua-ls uses `---@` to understand the types, so it throws an error if the declaration doesn't match what is expected.